definitions/service.md

```diff
@@ -1,17 +1,27 @@
 ---
 title: Service
-status: Feedback Appreciated
-category: concept
+status: Completed
+category: Concept
 ---
 ## Service
 
+### What it is
+
 A “Service” is a term with more meanings than virtually any other used in IT today. Traditionally a service represented a software system of which requests were made, as opposed to ‘fat client’ applications where all logic was contained in the program running on a user’s desktop. 
 
 In the context of Kubernetes the term service has a very specific meaning - something running in a Kubernetes cluster, that may be implemented by many instances, and that other workloads may wish to communicate with via an IP address. 
 
 In other contexts, a service might mean a web service (one that communicates via HTTP(S)), a data service (databases, message queues, caches) or microservices (applications embodying the service design pattern with a single responsibility).
 
-Services in Kubernetes allow workloads to communicate with an application without knowing how many instances of it are running, or the IP addresses of all of those instances. 
+### The problem it addresses
+
+A service in general, allows multiply system across the web to exchange information in a standardised and structured way.
```

Comment:
This isn't really a problem description. It describes what services do but not why they do what they do.

I'd only focus on the traditional definition. We already pointed to the k8s docs above and it feels too technical for this glossary.